MicroPython/src/cloud/AWS_cloud.py

```diff
@@ -16,21 +16,22 @@ def __init__(self) -> None:
         # TODO: Any initialization at constructor for a given cloud?
         pass
 
-    def device_configuration(self, data: dict) -> int:
+    def device_configuration(self, data: list[dict]) -> int:
         """
         Configures device in the cloud. Function used as hook to web_app.
         :param data: parameters to connect to wifi.
         :return: Error code (0 - OK, 1 - Error).
         """
-        ssid = data['ssid']
-        password = data['password']
 
-        config.cfg.ssid = ssid
-        config.cfg.password = password
+        access_points = data
+
+        config.cfg.access_points = access_points
```

Comment:
We use two different variables with the same data. The Variable access_points is not needed anyway, as we can use "data" parameter directly.